lib/services/wallet_connect_service/default_wallet_connect_service.dart

```diff
@@ -25,10 +24,32 @@ import 'models/wallet_connect_session_status.dart';
 import 'wallet_connect_session.dart';
 import 'wallet_connect_session_delegate.dart';
 
+//  final _keyValueService = get<KeyValueService>();
+//   final _accountService = get<AccountService>();
+//   final _connectionFactory = get<WalletConnectionFactory>();
+//   final _remoteNotificationService = get<RemoteNotificationService>();
+//   final _queueServce = get<WalletConnectQueueService>();
+//   final _transactionHandler = get<TransactionHandler>();
+//   final _localAuthHelper = get<LocalAuthHelper>();
```

Comment:
You can probably just remove these since we have the git history.

Reply:
oops. will remove this